### ISSUE TITLE
Long filenames display

### ISSUE THREAD

**Issue by kadowa** [kadowa/omp-vgwort-plugin#3]
Bei langen Dateinamen wird die Tabelle abgeschnitten:

![filenames_vgwort](https://cloud.githubusercontent.com/assets/8066366/12323389/1728c7be-babb-11e5-8f0e-f773e1ef5e4d.png)

Ich schlage vor, lange Dateinamen gekürzt anzuzeigen und versuche, eventuell den Private code unter dem Public anzuzeigen. Generell ist das aber nicht nur ein Anzeigeproblem für das VG-Wort-Plugin:

![file_names](https://cloud.githubusercontent.com/assets/8066366/12323400/3c7c6250-babb-11e5-989c-d4aeea7839a7.png)

Vielleicht sollten wir für auch für OMP einen Best-Practice-Guide erstellen und aufnehmen, dass Dateinamen eine gewisse Länge nicht überschreiten sollten (oder in diesem Fall zumindest Whitespace enthalten, damit umgebrochen werden kann). Die Dateinamen sind nur OMP-intern, der tatsächliche Name im Dateisystem ist anders, d.h. Whitespace oder Sonderzeichen im Namen sind kein Problem.

**Comment by withanage** [comment]
Ja, die Datei Länge sollte  am besten eine maximale Länge nicht überschreiten, damit noch in kleinere Bildschirmen angezeigt werden. Morgen kann man das bestimmen lassen,

**Comment by kadowa** [comment]
Aktueller Fix:

![filename_fix](https://cloud.githubusercontent.com/assets/8066366/12326027/92b63a3e-bace-11e5-950e-b1f87f4e97c3.png)
